src/renault_api/renault_vehicle.py

```diff
@@ -578,6 +578,28 @@ async def set_charge_stop(self) -> models.KamereonVehicleChargingStartActionData
             ),
         )
 
+    async def start_horn_lights(self, target: str):
```

Comment:
Please do not use this method to share processing.
Renault has a big habit of breaking things - so it's better to have duplicate code than trying to reuse code.